protocol-units/execution/suzuka/executor/src/v1.rs

```diff
@@ -225,7 +248,9 @@ mod opt_tests {
         ));
         let txs = ExecutableTransactions::Unsharded(vec![tx]);
         let block = ExecutableBlock::new(block_id.clone(), txs);
-        executor.execute_block(FinalityMode::Opt, block).await?;
+        let commitment = executor.execute_block(FinalityMode::Opt, block).await?;
+
+        // TODO: test the commitment
```

Comment:
Is this TODO outside the scope of this PR? Maybe lets make a tracking ticket . issue.

Reply:
I've added some checks in https://github.com/movementlabsxyz/movement/pull/58/commits/fae30bec917049a87ce0a577ff679f550f005f07
Not testing the commitment hash itself, it would require replicating much of the internal code.